tests/api/features/steps/boards_steps.py

```diff
@@ -0,0 +1,56 @@
+from logging import exception
+from behave import given, when, then, step
+from jsonschema import validate
+from main.core.rest.request_manager import RequestManager
+
+from main.trello.api.boards_manager import BoardsManager
+
+
+request_manager = RequestManager()
```

Comment:
FYI:
environment.py already creates an instance of request manager that's available for all features in `context.request_manager`